### PR TITLE
Add FakeConvert functional test

### PR DESCRIPTION
**Pull request by CailiWuIntel**
### Details:
Add FakeConvert functional test.

### Tickets:
 - *E-106805*

### CODE REVIEW

**Comment by YangleiZouIntel** [comment]
@ArtemySkrebkov Hi Artemy, could you please help review and merge this PR? Thanks.

File: src/tests/functional/plugin/shared/include/single_op_tests/fake_convert.hpp [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+//
+
+#pragma once
+
+#include "shared_test_classes/single_op/fake_convert.hpp"
+
+namespace ov {
+namespace test {
+
+TEST_P(FakeConvertLayerTest, Inference) {
```
Comment (ArtemySkrebkov) [inline src/tests/functional/plugin/shared/include/single_op_tests/fake_convert.hpp:12]: Is it instantiated somewhere, so we can check it passing?

**Review by ArtemySkrebkov — COMMENTED** [review]
@iefode @sbalandi 

can you please review as well?

File: src/tests/test_utils/common_test_utils/include/common_test_utils/node_builders/fake_convert.hpp [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+#pragma once
+
+#pragma once
+
+#include "openvino/core/node.hpp"
+
+namespace ov {
+namespace test {
+namespace utils {
+std::shared_ptr<ov::Node> make_fake_convert(const ov::Output<ov::Node>& in,
```
Comment (sbalandi) [inline src/tests/test_utils/common_test_utils/include/common_test_utils/node_builders/fake_convert.hpp:13]: Where is it used ?

File: src/tests/functional/plugin/shared/include/single_op_tests/fake_convert.hpp (hunk 1 above)
Comment (sbalandi) [reply src/tests/functional/plugin/shared/include/single_op_tests/fake_convert.hpp:12]: you can find instantiation for example by path openvino/src/plugins/[intel_cpu/intel_gpu/other plugin name...]/tests/functional/shared_tests_instances/single_layer_tests/

**Comment by CailiWuIntel** [comment]
Hi @ArtemySkrebkov @sbalandi, there are some changes in the current test plan, so this PR is not needed for now. I'll close it, thanks for your comments!

**Pull request closed without merging**